ads/google/a4a/utils.js

```diff
@@ -293,3 +310,83 @@ export function getCorrelator(win, opt_cid) {
   }
   return win.ampAdPageCorrelator;
 }
+
+/**
+ * Gets a relatively terse and controlled-format string version of the user
+ * agent.
+ *
+ * @param {!Window} win
+ * @return {string}
+ */
+export function userAgentString(win) {
```

Comment:
Oh right, of course.  I tend to just think of the bare ping URL as being recorded, but, of course, there's much more available.  Deleted.